### PR TITLE
Deal - more configurability

### PR DESCRIPTION
**Pull request by EdNoepel**
Let operators choose which addresses for which they deal auctions.
Also added some comments and code cleanup.

### CODE REVIEW

File: auction_keeper/main.py [hunk 1]
```diff
@@ -184,6 +189,21 @@ def __init__(self, args: list, **kwargs):
 
         logging.basicConfig(format='%(asctime)-15s %(levelname)-8s %(message)s',
                             level=(logging.DEBUG if self.arguments.debug else logging.INFO))
+
+        # Configure account(s) for which we'll deal auctions
+        print(self.arguments.deal_for)
```
Comment (grandizzy) [inline auction_keeper/main.py]: this line can be removed

File: auction_keeper/main.py [hunk 2]
```diff
@@ -69,6 +70,9 @@ def __init__(self, args: list, **kwargs):
                             help="Do not take opportunities to create new auctions")
         parser.add_argument('--kick-only', dest='bid_on_auctions', action='store_false',
                             help="Do not bid on auctions")
+        parser.add_argument('--deal-for', type=str, nargs="+",
```
Comment (grandizzy) [inline auction_keeper/main.py]: would be good to have a startup sample in readme file for clarity

File: auction_keeper/main.py [hunk 3]
```diff
@@ -403,13 +435,21 @@ def check_all_auctions(self):
 
         self.logger.debug(f"Checked {self.strategy.kicks()} auctions in {(datetime.now() - started).seconds} seconds")
 
+    def check_for_bids(self):
```
Comment (grandizzy) [inline auction_keeper/main.py]: why is this function declared twice?

File: auction_keeper/main.py (hunk 2 above)
Comment (EdNoepel) [reply auction_keeper/main.py]: There's a whole new section in there for it; check line 247.

File: auction_keeper/main.py (hunk 3 above)
Comment (EdNoepel) [reply auction_keeper/main.py]: Bad merge.  Resolved; will retest before pushing.

File: README.md [hunk 4]
```diff
@@ -167,52 +167,7 @@ For some known Ubuntu and macOS issues see the [pymaker](https://github.com/make
...
-                        Amount of Dai to keep in the Vat contract (e.g. 2000)
-  --keep-dai-in-vat-on-exit
-                        Retain Dai in the Vat on exit, saving gas when
-                        restarting the keeper
-  --keep-gem-in-vat-on-exit
-                        Retain collateral in the Vat on exit
-  --model MODEL         Commandline to use in order to start the bidding model
-  --debug               Enable debug output
-```
+Run `bin/auction-keeper` without arguments to see an up-to-date list of arguments and usage information.
```
Comment (Exef) [inline README.md]: I would instead propose to run `bin/auction-keeper -h`.

Comment (EdNoepel) [reply README.md]: done 👍

**Pull request merged**